specification/e-referrals-service-api.yaml

```diff
@@ -118,15 +118,15 @@ info:
     - by ‘Provider Organisations’ (currently)
     - for accessing endpoints [A004](#get-/STU3/CodeSystem/-codeSystemType-), [A005](#get-/STU3/ReferralRequest/-ubrn-), [A006](#get-/STU3/Binary/-attachmentLogicalID-), [A007](#post-/STU3/ReferralRequest/-ubrn-/$ers.generateCRI), [A024](#get-/STU3/CommunicationRequest/-ubrn-), [A025](#get-/STU3/Communication) and [A043](#post-/STU3/CommunicationRequest/-ubrn-/$ers.generateCRI) (currently)
 
-    For more information, please see [Network access](#api-description__network-access).
+    For more information, please see [Network access](#api-description--network-access).
 
     ### Healthcare worker, user-restricted access
 
     This access mode is [in production, beta](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#statuses). We are onboarding Partners to use it.
     
     This access mode allows you to use [all endpoints](#api-endpoints) . A human user must be present and authenticated.
     
-    For more information, please see [Network access](#api-description__network-access).
+    For more information, please see [Network access](#api-description--network-access).
 
```

Comment:
A bit picky as there are only 2 uses of it, but its probably better to move this to populate_paceholders and create [[NYPERLINK_NETWORK_ACCESS]]

Reply:
Done